core/kazoo_bindings/src/kazoo_bindings.erl

```diff
@@ -82,9 +96,25 @@
 -type kz_binding() :: #kz_binding{}.
 -type kz_bindings() :: [kz_binding()].
 
+-type kz_rt_options() :: kz_proplist().
+-type kz_rt_option() :: 'candidates' | 'matches'.
+
 -record(state, {bindings = [] :: kz_bindings()}).
 -type state() :: #state{}.
 
+-export_type([kz_responder/0
+             ,kz_responders/0
+             ,kz_binding/0
+             ,kz_bindings/0
+             ,candidates_fun/0
+             ,matches_fun/0
+             ,payload/0
+             ,map_results/0
+             ,fold_results/0
+             ,kz_rt_option/0
+             ,kz_rt_options/0
```

Comment:
How about dropping the `kz_` prefix? Exported types have `?MODULE:` prefix anyway